README.md

```diff
@@ -67,6 +67,10 @@ https://localhost:3000
 *[Please report issues and suggestions](https://github.com/ayoayco/astro-reactive-library/issues)*
 
 # Contributors
+<a href="https://github.com/ayoayco/astro-reactive-library/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=ayoayco/astro-reactive-library" />
+</a>
+
 1. [Ayo Ayco](https://ayco.io)
 2. [Jaspreet Singh Saini](https://jaspreetdesigns.netlify.app/)
 3. *[Add your name here!](https://github.com/ayoayco/astro-reactive-library/blob/main/CONTRIBUTING.md)*
```

Comment:
Hi @adetoye-dev ! Greate update!

Just one small adjustment. Let's remove these list of names to be replaced with the images for now so the contributors are always updated. :)

Reply:
Alright, I'll fix that 😊